docs/content/en/docs/clustermgmt/observability/cluster-verify.md

```diff
@@ -9,117 +9,36 @@ description: >
   How to verify an EKS Anywhere cluster is running properly
 ---
 
-**Verify an EKS Anywhere cluster from the target cluster**
-First, set the `KUBECONFIG` environment variable is set up for the target cluster (management or workload cluster) of choice:
+### Verify an EKS Anywhere cluster
+
+First, set the `KUBECONFIG` environment variable for your cluster:
 
 ```
 export KUBECONFIG=<path-to-cluster-kubeconfig>
 ```
 
-To verify that the cluster components are up and running, use the `kubectl` command to show that the pods are all running.
-There will be more or less pods depending on whether a management or workload cluster is targeted.
+To verify that the cluster components are up and running, use the `kubectl` command to show that the pods are all running. Below is the example but output may change as per the deployment. Please ensure that pods should be in running STATUS:
```

Comment:
Bit of a nit, but this repo is supposed to use https://sembr.org/ for this and the _index.md above